src/schemas/note.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod";
```

Comment:
we using [Yup](https://github.com/jquense/yup) for the validation form

Reply:
I don't think we've ever enforced a certain verification library

Reply:
Hey @AshleyDawson , what lib are we using for form validation? Because in the desktop-product I see zod and yap

Reply:
@zpawn I think the preference now is Zod